src/desktop/apps/artist/maybe_new_artist_page.ts

```diff
@@ -0,0 +1,11 @@
+// TODO: Read from ENV.
+const NEW_ARTIST_PAGE_THRESHOLD = 3
+
+module.exports = (_req, res, next) => {
+  console.log(res.locals.sd.NEW_ARTIST_PAGE)
```

Comment:
Stray loggy?

Reply:
yes, was just testing! Still need to switch to grab the const from ENV/config, will remove the log too.